### PR TITLE
bpo-13474 Add -m reference  in context of code execution

### PR DESCRIPTION
**Pull request by elusivenode**
https://bugs.python.org/issue13474

### CODE REVIEW

**Comment by JulienPalard** [comment]
@ncoghlan, ping

File: Doc/reference/executionmodel.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ The following are blocks: a module, a function body, and a class definition.
 Each command typed interactively is a block.  A script file (a file given as
 standard input to the interpreter or specified as a command line argument to the
 interpreter) is a code block.  A script command (a command specified on the
-interpreter command line with the '**-c**' option) is a code block.  The string
+interpreter command line with the '**-c**' option) is a code block.  A module run as a top level script (as module `__main__`) from the command line using a '**-m**' argument is also a code block. The string
```
Comment (ncoghlan) [inline Doc/reference/executionmodel.rst:25]: Double backticks on the markup

**Review by ncoghlan — CHANGES_REQUESTED** [review]
Looks good, but needs a markup fix (I'm on my phone, or I'd fix it directly)

**Comment by ncoghlan** [comment]
(I am puzzled as to why me-of-a-year-ago didn't just fix that. Sorry for the delay...)

**Pull request closed without merging**